usaspending_api/references/models.py

```diff
@@ -185,6 +186,18 @@ class Meta:
         db_table = 'office_agency'
 
 
+class FilterHash(models.Model):
+    filter_hash_id = models.AutoField(primary_key=True)
+    create_date = models.DateTimeField(auto_now_add=True, blank=True, null=True)
```

Comment:
I noticed that all the models seem to have null=true for create_data. I believe this is set because null=True is required bj Djago if the field is not unique.